### ISSUE TITLE
How to track mouseover and get column index and row index?

### ISSUE THREAD

**Issue by vingrad** [FancyGrid/FancyGrid#21]
How to track mouseover and get column index and row index? I need this info to display additional tipped with detailed Information about the data in cell.

**Comment by MikhailTatsky** [comment]
Here are doc links on this issue

If you need events specially for columns.
But they do not have information about row index
https://fancygrid.com/api/events/columnenter
https://fancygrid.com/api/events/columnleave

If you need events specially for rows.
https://fancygrid.com/api/events/rowenter
https://fancygrid.com/api/events/rowleave

Also there are special events for cells
https://fancygrid.com/api/events/cellenter
https://fancygrid.com/api/events/cellleave

In short all these events are similiar but with some small differences.
